src/utils/components/SidebarEditor/SidebarEditor.tsx

```diff
@@ -29,13 +30,15 @@ type SidebarEditorProps<Resource> = {
   onResourceUpdate?: (newResource: Resource) => Promise<Resource | void>;
   children: ReactNode | ReactNode[] | ((resource: Resource) => ReactNode);
   onChange?: (resource: Resource) => void;
+  pathsToHighlight?: string[];
 };
 
 const SidebarEditor = <Resource extends K8sResourceCommon>({
   children,
   resource,
   onResourceUpdate,
   onChange,
+  pathsToHighlight = ['spec.template.spec.domain.devices.disks', 'spec.template.spec.volumes'],
```

Comment:
Just curious: what is the reason of setting exactly these paths as the default values for the editor? Thanks, Ugo!

Reply:
PATHS_TO_HIGHLIGH.DEFAULT